.github/workflows/reuse-lint.yaml

```
@@ -0,0 +1,26 @@
+# Copyright 2023 SECO Mind Srl
+# SPDX-License-Identifier: Apache-2.0
+
+name: reuse
+
+permissions:
+  contents: read
+
+on:
+  push:
+    branches:
+      - master
+      - release-*
+  pull_request:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: REUSE Compliance Check
+        uses: fsfe/reuse-action@v2
```

Add new line at end of file
```suggestion
        uses: fsfe/reuse-action@v2

```